pkg/sidecar/sidecar_linux.go

```diff
@@ -74,28 +74,23 @@ func handler(ctx context.Context, instance *Instance) error {
 		return err
 	}
 
-	// Wait for all the sidecars to enter the "network-initialized" state.
-	const netInitState = "network-initialized"
-	if _, err = instance.Writer.SignalEntry(ctx, netInitState); err != nil {
-		return fmt.Errorf("failed to signal network ready: %w", err)
-	}
+	ctx = sync.WithRunParams(ctx, &instance.RunEnv.RunParams)
```

Comment:
@Stebalien and each sync operation is contextualised through the `context.Context`.

Reply:
Nice!